cli/js/web/workers.ts

```diff
@@ -28,6 +28,72 @@ function decodeMessage(dataIntArray: Uint8Array): any {
   return JSON.parse(dataJson);
 }
 
+export interface PostMessageOptions {
+  transfer: object[];
+}
+export interface MessagePort extends EventTarget {
+  postMessage(message: any, transfer: object[]): void;
+  postMessage(message: any, options: PostMessageOptions): void;
+}
+export type MessageEventSource = MessagePort | Worker; // | WindowProxy
+export interface MessageEventInit extends __domTypes.EventInit {
+  data?: any;
+  origin?: string;
+  lastEventId?: string;
+  source?: MessageEventSource | null;
+  ports?: MessagePort[];
```

Comment:
We don't support ports right now, it should throw error when `ports` option is passed